docs/essentials/viewport.mdx

```diff
@@ -173,63 +123,33 @@ If you need, you can edit these on the shortcuts page.
 * Previous viewport: <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>v</kbd>
 * Reset viewport: <kbd>alt</kbd> + <kbd>control</kbd> + <kbd>v</kbd>
 
-<Callout variant="warning" icon="🆕">
+### Globals
 
-  **With the [`globals` API](#globals-api)**
+This module contributes the following globals to Storybook, under the `viewport` namespace:
 
-  ### Globals
+#### `value`
 
-  This addon contributes the following globals to Storybook, under the `viewport` namespace:
-
-  #### `value`
-
-  Type: `string`
-
-  When set, the viewport is applied and cannot be changed using the toolbar. Must match the key of one of the [available viewports](#options).
+Type: `string`
 
-  #### `isRotated`
+When set, the viewport is applied and cannot be changed using the toolbar. Must match the key of one of the [available viewports](#options).
 
-  Type: `boolean`
+#### `isRotated`
 
-  When true the viewport applied will be rotated 90°, e.g. it will rotate from portrait to landscape orientation.
+Type: `boolean`
 
-</Callout>
+When true, the viewport applied will be rotated 90°, e.g., from portrait to landscape orientation.
 
 ### Parameters
 
-This addon contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `viewport` namespace:
-
-#### `defaultOrientation`
-
-Type: `'portrait' | 'landscape'`
-
-Default: `'portrait'`
-
-Specifies the default orientation used when viewing a story. Only available if you haven't enabled the [globals API](#customizing-viewports-with-globals).
-
-#### `defaultViewport`
-
-Type: `string`
-
-Specifies the default viewport used when viewing a story. Must match a key in the [`viewports`](#viewports) (or [`options`](#options)) object.
+This module contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `viewport` namespace:
 
 #### `disable`
```

Comment:
TODO: Check that this is truly `disable` and not `disabled` (as it used to be).

Reply:
Confirmed: https://github.com/storybookjs/storybook/blob/next/code/core/src/viewport/types.ts#L43